### PR TITLE
Improve code quality for Donor class

### LINKED ISSUE #102: Improve code quality for Donor class
Some comments about code quality for Donor class has been detailed in PR #82

### PR DESCRIPTION
**Pull request by zzzlou**
Fixes #102

### CODE REVIEW

**Comment by zzzlou** [comment]
As per my comment in #82 , I did not make use of super.toString() in children classes of Person:

Because of the use of ToStringBuilder (note that this is different from Java's built-in StringBuilder), I think we cannot use super.toString() here for the following reasons:

the object being passed in will be different(Donor vs Person).
person.toString() ends with a suffix "}", it would not be elegant to directly do: person.toString() - "}" + "hours" + hours.toString() as doing this would make the use of ToStringBuilder meaningless.
So in my opinion the best approach to keep the code concise here is to keep it this way.
Let me know if you have better ways of doing this :)

**Comment by zzzlou** [comment]
To be refined

File: src/main/java/seedu/address/model/person/Donor.java [hunk 1]
```diff
@@ -51,17 +50,12 @@ public boolean equals(Object other) {
...
         }
 
-        seedu.address.model.person.Donor otherDonor = (seedu.address.model.person.Donor) other;
-        return name.equals(otherDonor.name)
-                && phone.equals(otherDonor.phone)
-                && email.equals(otherDonor.email)
-                && address.equals(otherDonor.address)
-                && tags.equals(otherDonor.tags)
-                && donatedAmount.equals(otherDonor.donatedAmount);
+        seedu.address.model.person.Donor otherDonor = (Donor) other;
```
Comment (Justin-Yeo) [inline src/main/java/seedu/address/model/person/Donor.java:57]: I think we can import it at the top! Will allow for code to look cleaner 😃

**Review by Justin-Yeo — CHANGES_REQUESTED** [review]
Looks good! Just one small change to improve code readability 👍

File: src/main/java/seedu/address/model/person/Donor.java (hunk 1 above)
Comment (zzzlou) [reply src/main/java/seedu/address/model/person/Donor.java]: Oops, I missed this part. Fixed it!

**Review by seah-minlong — APPROVED** [review]
LGTM

**Review by Justin-Yeo — APPROVED** [review]
LGTM!

**Pull request merged**